bootstrap.go

```diff
@@ -552,6 +557,10 @@ func upgradeToVer16(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.stats_histograms ADD COLUMN `cm_sketch` blob", infoschema.ErrColumnExists)
 }
 
+func upgradeToVer17(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.user MODIFY User CHAR(32)")
```

Comment:
Should we also modify the create table statement of mysql.user?

Reply:
Done